docs/components/empty-state.md

```diff
@@ -249,6 +318,34 @@ Slot for providing custom icon.
 </KEmptyState>
 ```
 
+### image
+
+Instead of an icon, you can provide a custom image through the `image` slot.
+
+<KEmptyState
+  message="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla nec purus feugiat, molestie ipsum et, consequat nibh."
+  title="Empty State With Image"
+>
+  <template #image>
+    <img src="https://picsum.photos/640/300">
+  </template>
+</KEmptyState>
+
+```html
+<KEmptyState
+  message="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla nec purus feugiat, molestie ipsum et, consequat nibh."
+  title="Empty State With Image"
+>
+  <template #image>
+    <img src="https://picsum.photos/640/300">
+  </template>
```

Comment:
Should this be a slot, or a prop that accepts the image source string so that we have control over the image display? (Or is there a reason you went with a slot?)